Visage.FrontEnd/Visage.FrontEnd.Shared/wwwroot/output.css

```
@@ -39,7 +39,7 @@ html,
 :host {
   line-height: 1.5;
   /* 1 */
-  -webkit-text-size-adjust: 100%;
+  -text-size-adjust: 100%;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)